app/assets/javascripts/spaces.coffee

```diff
@@ -0,0 +1,14 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
+
+# "Selectize" an element by applying the Selectize js library
+# http://brianreavis.github.io/selectize.js/
+Spaces.selectizeElement = (element) ->
+	selectize_control = $(element).selectize 
+						 	maxItems: 5
+						 	delimiter: ','
+						 	persist: false
+						 	create: (input)  ->
+					 			value: input
+					 			text: input
```

Comment:
like before this should be in a Controller.  You could very easily just do `class Spaces.SpacesController extends Spaces.ViewController` in this file, and then add the `selectizeElement` method to that.

The big picture is getting away from global namespace.  Rails makes it seem like these coffeescript files are scoped according to the controller, but in reality it's all mashed into a single minified JS file that is sent down with all pages.